src/renderer/pages/settings/modules/about-settings.js

```
@@ -0,0 +1,485 @@
+
+  // 이벤트 리스너 등록
+  window.addEventListener('download-progress', progressListener);
+
+  // 업데이트 다운로드
+  window.settings.downloadUpdate()
+    .then(result => {
+      window.settings.log.info('업데이트 다운로드 결과:', result);
+
+      if (result.success) {
+        // 다운로드 성공
+        if (updateMessage) {
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)